src/tweenable.rs

```diff
@@ -791,10 +791,17 @@ impl<T> Tweenable<T> for Tween<T> {
         // If completed at least once this frame, notify the user
         if times_completed > 0 {
             if let Some(user_data) = &self.event_data {
-                events.send(TweenCompleted {
+                let event = TweenCompleted {
                     entity,
                     user_data: *user_data,
-                });
+                };
+
+                // send regular event
+                events.send(event);
+
+
```

Comment:
Can you please remove that extra blank line, which will also re-trigger a CI against the latest version (with Bevy 0.15 support)? Thanks